### PR TITLE
nukie jumpsuits and skirts sensors off

### LINKED ISSUE #9918: Nukies spawn with suit sensors on
## Description
Nukies spawn with suit sensors on. You want to remove suit sensors off nukies or make them detectable only with dedicated device nukie medic would get.

**Reproduction**
- forcepreset Nukeops
- Check out your randomized suit sensor setting

**Screenshots**
nukie_suit_sensors

### PR DESCRIPTION
**Pull request by JustinTrotter**
## About the PR 
Fixes #9918

**Changelog**

:cl:
- fix: nukies no longer have the chance to spawn with their suit sensors on

### CODE REVIEW

**Review by Elijahrane — APPROVED** [review]

**Review by juliangiebel — APPROVED** [review]

**Review by Veritius — APPROVED** [review]

**Comment by metalgearsloth** [comment]
Should they just not have the suitsensor component?

**Comment by JustinTrotter** [comment]
> Should they just not have the suitsensor component?

They inherit the suit sensor from the base uniform component.

**Comment by Emisse** [comment]
create a new parent without the sensors since this will likely be needed for other antag suits

File: Resources/Prototypes/Entities/Clothing/Uniforms/base_clothinguniforms.yml [hunk 1]
```diff
@@ -29,18 +17,32 @@
 
 - type: entity
   abstract: true
-  parent: ClothingWithSuitSensor
+  parent: UnsensoredClothingUniformBase
+  id: UnsensoredClothingUniformSkirtBase
+  description: Not legal in Japan
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Uniforms/base_clothinguniforms.yml:22]: bruh

Comment (JustinTrotter) [reply Resources/Prototypes/Entities/Clothing/Uniforms/base_clothinguniforms.yml]: Aw.

Gone.

**Pull request merged**